### ISSUE TITLE
Using this plugin when profiling results in invalid JSON

### ISSUE THREAD

**Issue by dnasir** [dividab/tsconfig-paths-webpack-plugin#6]
Here is an extract of my `stats.json` file when I run `webpack --profile --json > stats.json` 

```json
tsconfig-paths-webpack-plugin: Using config file at C:\projects\myproject\tsconfig.json
{
  "errors": [],
  "warnings": [],
  "version": "3.10.0",
  "hash": "ee90be31c4387bf2ebe1",
  "time": 40101,
  "publicPath": "",
  "assetsByChunkName": {}
}
```

I suspect this issue is due to line 87 in `plugin.ts` where the plugin logs a warning when it runs. I don't think this is a good idea as the `>` symbol in command outputs all `stdout` to the file specified after the symbol.

**Comment by jonaskello** [comment]
Should be fixed in 1.4.0, please give it a try!

**Comment by dnasir** [comment]
I can confirm that this issue has been resolved. Thanks!
